### PR TITLE
922 cursor

### PR DESCRIPTION
**Pull request by rsumilang**

### CODE REVIEW

**Comment by efjacobson** [comment]
@rsumilang seems like a weird branch situation here...quite a few unrelated file changes

File: src/plugins/blocksmith/utils/index.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ import isFirstListBlockFn from './isFirstListBlock';
 import isLastListBlockFn from './isLastListBlock';
 import loadFacebookSDKFn from './loadFacebookSDK';
 import loadTwitterSDKFn from './loadTwitterSDK';
+import moveSelectionToEndOfBlockFn from './moveSelectionToEndOfBlock';
```
Comment (efjacobson) [inline src/plugins/blocksmith/utils/index.js:36]: @rsumilang there is a util for this already, `selectBlock`, use it with `selectionTypes.END` to do the same as `moveSelectionToEndOfBlock`

**Comment by arunkarnati** [comment]
@rsumilang - to fix the weird changeset issue you can either rebase commits onto master or just check out a new branch to apply the changes (since the changeset is not that bad)

Also, I don't think there is any need for a `release-next` branch in a package as we don't batch the changes for one big release.

**Comment by rsumilang** [comment]
Moved changes to `922-cursor-new`.

**Pull request closed without merging**